app/components/combined_system_message/combined_system_message.test.js

```diff
@@ -12,10 +12,14 @@ import {Posts} from 'mattermost-redux/constants';
 
 import CombinedSystemMessage from './combined_system_message';
 
+/* eslint-disable max-nested-callbacks, no-console */
+
 describe('CombinedSystemMessage', () => {
+    function emptyFunc() {} // eslint-disable-line no-empty-function
```

Comment:
there is and empty function in `utils/general` you can use that one

Reply:
Got it, thanks!